modules/uploads/scripts/get_file_links.ts

```diff
@@ -0,0 +1,72 @@
+import { ScriptContext } from "../_gen/scripts/get_file_links.ts";
```

Comment:
"link" is ambiguous. Should be something like "get_public_file_urls" or something like that.